packages/web/src/components/data-entry/ContextualMenu.tsx

```diff
@@ -122,11 +123,14 @@ export const ContextualMenu = <FormValues extends FormikValues = FormikValues>(
     onSubmit,
     error,
     errorMessage,
+    previewOverride,
     ...formikProps
   } = props
   const [isMenuOpen, toggleMenu] = useToggle(false)
 
-  const preview = (
+  const preview = previewOverride ? (
+    previewOverride(toggleMenu)
+  ) : (
     <Tile onClick={toggleMenu} className={styles.root} elevation='flat'>
```

Comment:
I can see that too, or maybe the reverse where contextual menu is the special one that renders the tile? I think I prefer having renderPreview or renderSomething as the prop instead of previewOverride though